terraform/dev.tf

```diff
@@ -0,0 +1,179 @@
+# TODO: All of the code in this file will need to be moved to a module that can be reused for other environments and 
```

Comment:
i went ahead and launched this as is so i could go ahead and get a demo site up. ultimately we want this to be a reusable module because it has the potential to serve pretty much any static site.

so in the short term we will definitely need a production site at least. but there's no reason we can't reuse this same code to launch infrastructure for any number of static frontend apps and their various development, staging and production enviornments